README.en.md

```diff
@@ -16,59 +16,59 @@
    
 ## An Ideal Music Player
 
-A simple design of just playing the mp3 files under selected directory.
+Toa has a simple design, built to streamline it's purpose: playing MP3s from a directory.
 
-Toa has just enough UI to enjoy music, so Toa won't occupy your screen.
+Toa has just enough UI to enjoy music, so it won't take up unnecessary screen space.
 
-It also has the beautiful exterior of the flat design like below:
+It utilizes flat design, like so:
```

Comment:
Thank you, but I'd rather prefer "as below" to "like so".

Reply:
I thought it might be better to remove directionality.  Either way: changed!